### PR TITLE
feat: release 3.7.0

### PR DESCRIPTION
**Pull request by ZurabDev**
### Features

- feat(components/input-multi-select): added transformer support #938 [MR](https://github.com/zyfra/Prizm/pull/1113)
- feat(icons): divided location and less #1047 [MR](https://github.com/zyfra/Prizm/pull/1114)
- feat(components/switcher): ability to change appearance #978 [MR](https://github.com/zyfra/Prizm/pull/1097)
- feat(documentation/theme-guide): theme guide for developers #979 [MR](https://github.com/zyfra/Prizm/pull/1060)

### Bug fixes

- fix(doc/dropdown-host): example with range components #893 [MR](https://github.com/zyfra/Prizm/pull/1096)
- fix(components/file-upload): progress status translations for file upload #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
- fix(components/dropdown-host): fix close logic #1074 [MR](https://github.com/zyfra/Prizm/pull/1090)
- fix(components/table): fix error when pass not observable value #1068 #1084 [MR](https://github.com/zyfra/Prizm/pull/1094)
- fix(doc/input-chips): fix doc with print chips value #572 [MR](https://github.com/zyfra/Prizm/pull/1095)
- fix(doc/dropdown-host): example with range components #893 [MR](https://github.com/zyfra/Prizm/pull/1096)

### Refactor

- refactor(components/file-upload): functions for file size and uploading status are moved to pipes #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
- refactor(components/file-upload): deprecated modules replaced by standalones i File Upload Component #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
- refactor(components/file-upload): translations for uploading status added [MR](https://github.com/zyfra/Prizm/pull/1032)
- refactor(components/input-chips): changed dependencies to actual standalone

### CODE REVIEW

File: apps/doc/src/app/about-prizm/changelog/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+## [1.11.0, 2.8.0, 3.7.0](https://github.com/zyfra/Prizm) (01-12-2023)
+
+### Features
+
+- feat(components/input-multi-select): added transformer support #938 [MR](https://github.com/zyfra/Prizm/pull/1113)
+- feat(icons): divided location and less #1047 [MR](https://github.com/zyfra/Prizm/pull/1114)
+- feat(components/switcher): ability to change appearance #978 [MR](https://github.com/zyfra/Prizm/pull/1097)
+- feat(documentation/theme-guide): theme guide for developers #979 [MR](https://github.com/zyfra/Prizm/pull/1060)
+- refactor(components/file-upload): funtions for file size and uploading status are moved to pipes #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
+- refactor(components/file-upload): deprecated modules replaced by standalones i File Upload Component #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
```
Comment (ickisIckis) [inline apps/doc/src/app/about-prizm/changelog/CHANGELOG.md:14]:
```suggestion
- refactor(components/file-upload): deprecated modules replaced by standalones in File Upload Component #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
```

File: apps/doc/src/app/about-prizm/changelog/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,28 @@
...
+- refactor(components/file-upload): translations for uploading status are moved from fileUploadOptions to translations #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
```
Comment (ickisIckis) [inline apps/doc/src/app/about-prizm/changelog/CHANGELOG.md]:
```suggestion
- refactor(components/file-upload): translations for uploading status added  #931 [MR](https://github.com/zyfra/Prizm/pull/1032)
```

**Review by ickisIckis — APPROVED** [review]
LGTM

**Review by alexhawkins94 — COMMENTED** [review]
На 14м Ангуляре у компонента Dropdown в примере Nested dropdown во внутреннем ипунте время обрезается многоточием
На 15 и 16 такого не наблюдаю, масштаб одинаковый 
chrome_ZYVskEWXrU

**Pull request merged**